podman/api/tar_utils.py

```diff
@@ -1,22 +1,22 @@
 """Utility functions for working with tarball's."""
-import os
 import pathlib
 import random
 import shutil
-import sys
 import tarfile
 import tempfile
 from fnmatch import fnmatch
 from typing import BinaryIO, List, Optional
 
+import sys
+
 
 def prepare_dockerignore(anchor: str) -> List[str]:
     """Read .dockerignore and return exclude list."""
-    file = os.path.join(anchor, ".dockerignore")
-    if not os.path.exists(file):
+    ignore = pathlib.Path(anchor) / ".dockerignore"
```

Comment:
There's also a `.containerignore` which ought to be searched for/used too.  Similar to Dockerfile/Containerfile, at the moment, same thing under the covers.

Reply:
@TomSweeneyRedHat If both exist, should one have higher precedence over the other or should I merge them?

Reply:
`.containerignore` wins if found and `.dockerignore` is not used regardless of presence.  If no `.containerignore`, then check for and use `.dockerignore` if that is found.


 vi ./vendor/github.com/openshift/imagebuilder/builder.go
```
// ParseDockerIgnore returns a list of the excludes in the .containerignore or .dockerignore file.
func ParseDockerignore(root string) ([]string, error) {
        excludes, err := ParseIgnore(filepath.Join(root, ".containerignore"))
        if err != nil && os.IsNotExist(err) {
                excludes, err = ParseIgnore(filepath.Join(root, ".dockerignore"))
        }
        if err != nil && os.IsNotExist(err) {
                return excludes, nil
        }
        return excludes, err
}
```

Reply:
Thanks, easiest to implement.